www/class/CentreonLDAPAdmin.class.php

```diff
@@ -518,14 +598,16 @@ public function deleteConfiguration($configList = array())
      * @param array $configList
      * @return void
      */
-    public function setStatus($status, $configList = array())
+    public function setStatus($status, $configList = [])
     {
         if (count($configList)) {
-            $this->db->query(
-                "UPDATE auth_ressource 
-                SET ar_enable = '" . $this->db->escape($status) . "'
+            $statement = $this->db->prepare(
+                "UPDATE auth_ressource
+                SET ar_enable = :ar_enable
                 WHERE ar_id IN (" . implode(',', $configList) . ")"
```

Comment:
loop to bind values

Reply:
I did that in the previous commit but It can be really heavy if the is a lot of ressource to update

Reply:
It can't be heavy cause nobody has more than 10 LDAP configurations

Reply:
loop re added